### DISCUSSION TITLE
\bbalpha replaced by ff in equation

### DISCUSSION THREAD

**Discussion by baptiste** [quarto-dev/quarto-cli#3989] [Q&A] [answered]
I'm compiling a book where I use blackboard symbols with the `mathbbol` latex package as in this example,

````
---
title: "Untitled"
format: 
  pdf: 
    keep-tex: true
    include-in-header: 
       text: |
         \usepackage[bbgreekl]{mathbbol}
---

This is a Quarto book.

To learn more about Quarto books visit <https://quarto.org/docs/books>.

$$
\mathbf{E}^i= \bbalpha_j \mathbf{E}^j,
$$

````

Now something really strange happens when I run quarto render: the bbalpha symbol is replaced by ff

<img width="746" alt="Screenshot 2023-01-17 at 18 53 02" src="https://user-images.githubusercontent.com/18970/212821062-7bb59aa0-d52b-4322-a96e-34d47b2cd968.png">

I'm stumped, because running xelatex manually on the tex file produces the expected output!

<img width="867" alt="Screenshot 2023-01-17 at 18 59 24" src="https://user-images.githubusercontent.com/18970/212821276-96a8c03b-11a6-4877-9257-ee8f2e894971.png">

I haven't compiled this book for some time but I never noticed this problem before, so it might be a recent change in Quarto? I installed it this morning 
```
quarto --version
99.9.9
```
(I'm guessing it's actually [v1.3.113](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.3.113), at commit 63175b359468910ccc5459ea28c10c7cdae8fcae)

Any idea as to what could be going on?

**Comment by baptiste** [comment]
Ah, nevermind, must be something to do with tinytex on my machine, since setting `latex-tinytex: false` solves it.

**Comment by cderv** [comment, marked as answer]
I was curious about this issue really. From my test, it is possible that you get two installation on your system (TinyTeX and another) and by deactivating TinyTeX you used the other one. 

What I found is that I got the [package **bbold**](https://ctan.org/tex-archive/fonts/bbold) missing with the required fonts for [**mathbbol** ](https://ctan.org/pkg/mathbbol)
[Doc of **mathbbol** ](https://mirrors.rit.edu/CTAN/macros/latex/contrib/jknappen/mathbbol.rme) says 
> The mathbbol package exploits the bbold-fonts and the stmaryrd-fonts designed by Alan Jeffrey and Jeremy Gibbons.
> Please find the bbold and stmary fonts on your nearest CTAN archive in directory tex-archive/fonts/bbold.

So I assume I need to install and I did 
````powershell
tlmgr install bbold
````

Font got installed, and then it was working. 

Sharing in case it helps

**Reply by baptiste** [reply]
That does seem to be the issue, thanks. Odd for LaTeX to not complain when something is missing and just replace glyphs!

**Reply by cderv** [reply]
> Odd for LaTeX to not complain when something is missing and just replace glyphs!

Yes I agree. It may be complaining in the log but not in an "error way" as usually when we detect missing fonts, we try to install them. 

🤷
